locales/fr.lua

```diff
@@ -20,8 +20,10 @@ local Translations = {
     info = {
         ["timeleft"] = "Il vous reste encore... %{JAILTIME} mois",
         ["lost_job"] = "Vous êtes au chômage",
+        ["job_interaction"] = "[E] Electricity Work", -- This needs to be translated
```

Comment:
If you wanna have French translation: `[E] Travailler sur l'électricité` (so we don't open another PR for 1 line)